package.json

```diff
@@ -40,9 +40,13 @@
     "prepublishOnly": "npm run build",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "test": "jest --projects jest.unit.config.js --coverage",
-    "coverage:visual": "jest --projects jest.visual.config.js --coverage"
-  },
+    "test": "jest --coverage --coverageDirectory=coverage/unit",
+    "coverage:visual": "jest --coverage --coverageDirectory=coverage/visual-regression --collectCoverageFrom='[\"src/components/**\"]' --testMatch=\"<rootDir>/tests/storyshot.test.ts\"",
+    "coverage:clean": "rm -rf .nyc_output && rm -rf coverage",
+    "coverage:merge": "istanbul-merge --out coverage/merged/coverage-final.json ./coverage/unit/coverage-final.json  ./coverage/visual-regression/coverage-final.json",
```

Comment:
Is the coverage merging supposed to be part of this PR? If I recall, there was a separate PR for that.

Reply:
Yes, because the combined-coverage PR needs to use coverage:merge scripts from main in github action coverage:base which is testing the main branch. So the scripts need to be merged in for the action to work.